### PR TITLE
LPD-16432 - [P3] Automate LPP-47573

### PR DESCRIPTION
**Pull request by G-Sa**
Related to :[LPD-16432](https://liferay.atlassian.net/browse/LPD-16432)

### CODE REVIEW

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java [hunk 1]
```diff
@@ -437,6 +437,14 @@ public void testObjectEntry1toMObjectRelatedModelsProviderImpl()
 			_objectRelationship.getObjectRelationshipId(),
 			objectEntry8.getObjectEntryId());
 
+		// Remove object entry to assert empty one to many relationship
```
Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java:440]: @G-Sa I did not understand the `to assert empty one to many relationship`. I did not see any `assert` here.

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java [hunk 2]
```diff
@@ -437,6 +437,14 @@ public void testObjectEntry1toMObjectRelatedModelsProviderImpl()
...
+
+		_updateObjectEntry(
+			objectEntry7.getObjectEntryId(),
+			HashMapBuilder.<String, Serializable>put(
+				_relationshipObjectField.getName(), 0
```
Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java:445]: @G-Sa In the bug ticket description we have:

> 5. Edit the entry for B and try to remove the relationship by leaving it empty.

Shouldn't we pass an empty string instead?

Comment (G-Sa) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java:445]: What i understand from that line is that we should remove the relationship between the two entries, as in allowing the object entry for B to not be related to anything anymore, thats why i edited the field assigning it to "nothing"
@carolmariaabb

Comment (carolmariaabb) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java:445]: Ok, thanks @G-Sa

**Comment by G-Sa** [comment]
ci:test:sf

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by G-Sa** [comment]
ci:forward:force

**Pull request closed without merging**